AnnotatoSharedLibrary/Sources/AnnotatoSharedLibrary/Models/Annotation.swift

```diff
@@ -2,17 +2,29 @@ import Foundation
 import CoreGraphics
 
 public final class Annotation: Codable {
-    public let id: UUID?
+    public let id: UUID
     public private(set) var origin: CGPoint
     public private(set) var width: Double
     public let ownerId: String
     public let documentId: UUID
 
-    public required init(origin: CGPoint, width: Double, ownerId: String, documentId: UUID, id: UUID? = nil) {
-        self.id = id
+    public required init(
+        origin: CGPoint,
+        width: Double,
+        ownerId: String,
+        documentId: UUID,
+        id: UUID? = nil
```

Comment:
```suggestion
        id: UUID? = UUID()
```
Since this is essentially what we are doing